ScrapperCron/Program.cs

```diff
@@ -1,49 +1,27 @@
-﻿using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Hosting;
-using ScrapperCron.Services;
-using System;
-using System.Threading.Tasks;
+﻿using Microsoft.Extensions.Hosting;
+using ScrapperCron.Infrastructure;
+using Serilog;
 
-namespace Cron_BolsaDeTrabajo
+namespace ScrapperCron
```

Comment:
Typo, a scrapper and a scraper are not the same thing

Reply:
Done!